js/views/userPageVw.js

```diff
@@ -573,11 +574,14 @@ module.exports = Backbone.View.extend({
     });
     this.subViews.push(this.itemList);
 
-    new window.List('searchStore', {valueNames: ['js-searchTitle'], page: 1000});
+    if (model.length) {
```

Comment:
I'm getting Uncaught TypeError: Cannot read property 'length' of undefined here, I'm not sure why. I'm testing it on a page that returns a model array with one item in it.